### PR TITLE
build docs on push to main

### PR DESCRIPTION
**Pull request by callumrollo**
This will automatically build the documentation site when a push is made to the main branch, keeping the docs up to date

### CODE REVIEW

**Review by vturpin — APPROVED** [review]
Even though I do not understand the role of this two lines ;) I agree on the principle of automation.

File: .github/workflows/adoc_build.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name: Export and publish document
 
 on:
-  workflow_dispatch:
```
Comment (castelao) [inline .github/workflows/adoc_build.yml:4]: I suggest keeping this one together with push. I found it useful when developing to be able to recompile it manually. By keeping both, we expect to have the auto mode on daily activities.

**Review by castelao — CHANGES_REQUESTED** [review]

**Comment by castelao** [comment]
@callumrollo. thanks !

@vturpin , you raised a very important issue, the versioning control. Since we don't have a stable full version yet, I think it is OK to keep moving along as we've been doing. We need to move efficiently now. Once we are getting closer to a stable version, we shall define a clear protocol. There are many ways of doing that. Let's save this discussion for later, but don't forget about it.

**Comment by callumrollo** [comment]
Yes, some versioning will ikely be needed down the line. Especially once the format is in production and we may have multiple versions. This should do for now though

**Review by castelao — APPROVED** [review]

**Pull request merged**